syndeo-template/src/main/java/com/google/cloud/syndeo/SyndeoTemplate.java

```diff
@@ -68,6 +69,29 @@ public interface Options extends PipelineOptions, DataflowPipelineOptions {
     void setJsonSpecPayload(String jsonSpecPayload);
   }
 
+  public static final Map<String, Set<String>> SUPPORTED_URNS =
+      Map.of(
+          // Old names:
+          "bigquery:read",
+          Set.of("table"),
+          "kafka:read",
+          Set.of(),
+          "bigtable:write",
+          Set.of("instanceId", "tableId", "keyColumns", "projectId", "appProfileId"),
+          "schemaIO:bigquery:write",
+          Set.of("table", "createDisposition", "writeDisposition"),
+          // New names:
+          "beam:schematransform:org.apache.beam:bigquery_storage_write:v1",
+          Set.of("instanceId", "tableId", "keyColumns", "projectId", "appProfileId"),
+          "beam:schematransform:org.apache.beam:bigquery_storage_read:v1",
+          Set.of("table"),
+          "beam:schematransform:org.apache.beam:kafka_write:v1",
+          Set.of(),
+          "beam:schematransform:org.apache.beam:kafka_read:v1",
+          Set.of(),
+          "syndeo:schematransform:com.google.cloud:bigtable_write:v1",
```

Comment:
nit: it might be easier to read these URNs if a blank line is added after each `Set.of` line

Reply:
the formatting is decided by spotless, not by me.